data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2159,6 +2217,91 @@ spec:
+                      not define their own platform configuration.
+                    properties:
+                      memory:
+                        description: Memory defines the memory in GB for the instance.
+                        type: string
+                      procType:
+                        description: ProcType defines the processor sharing model
+                          for the instance. Must be one of {capped, dedicated, shared}.
+                        enum:
+                        - capped
+                        - dedicated
+                        - shared
+                        type: string
+                      processors:
+                        description: Processors defines the processing units for the
+                          instance.
+                        type: string
+                      sysType:
+                        description: SysType defines the system type for instance.
+                        type: string
+                      volumeIDs:
+                        description: VolumeIDs is the list of volumes attached to
+                          the instance.
+                        items:
+                          type: string
+                        type: array
+                    type: object
+                  powervsResourceGroup:
+                    description: PowerVSResourceGroup is the resource group for creating
+                      Power VS resources.
+                    type: string
+                  pvsNetworkName:
+                    description: PVSNetworkName specifies an existing network within
+                      the Power VS Service Instance.
+                    type: string
+                  region:
+                    description: Region specifies the IBM Cloud region where the cluster
+                      will be created.
+                    type: string
+                  serviceInstanceID:
+                    description: ServiceInstanceID is the ID of the Power IAAS instance
+                      created from the IBM Cloud Catalog
+                    type: string
+                  subnets:
+                    description: Subnets specifies existing subnets (by ID) where
+                      cluster resources will be created.  Leave unset to have the
+                      installer create subnets in a new VPC on your behalf.
+                    items:
+                      type: string
+                    type: array
+                  userID:
+                    description: UserID is the login for the user's IBM Cloud account.
+                    type: string
+                  vpc:
+                    description: VPC is a VPC inside IBM Cloud. Needed in order to
+                      create VPC Load Balancers.
+                    type: string
+                  vpcRegion:
+                    description: Zone in the region used to create VPC resources.
```

Comment:
is this vpcRegion or vpcZone...there seems to be some inconsistency here with the naming

Reply:
region. that looks like a copy-paste problem in the description